var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -125,6 +133,18 @@ def install(self, spec, prefix):
         sh = which('sh')
         sh('./autogen.sh')
 
+        if self.get_arch() == 'ppc64le':
+            # update the config.sub/guess files
+            with working_dir("config"):
+                # get new config.guess and config.sub files
+                print 'Backing up existing config.[sub|guess] files\n'
+                subprocess.call("mv config.sub config.sub.orig", shell=True)
+                subprocess.call("mv config.guess config.guess.orig",
```

Comment:
Just use the `move` command, an alias for `shutil.move`, which is already in the package's global scope.